2016/ideas-list-swc.md

```diff
@@ -255,16 +262,16 @@ Other: please use our [mailing list][].
 Thanks to @xuf12 for the initial idea behind this project and for @wking for
 providing this idea for GSOC'15.
 
-  [Software Carpentry]: https://software-carpentry.org/
-  [gsoc-issues]: https://github.com/numfocus/gsoc/issues
-  [swc]: https://github.com/swcarpentry/workshop-template/tree/gh-pages/setup
-  [swc-install]: https://github.com/wking/swc-setup-installation-test
-  [Python]: https://www.python.org/
-  [Django]: https://www.djangoproject.com/
-  [AMY]: https://github.com/swcarpentry/amy
-  [Flask]: http://flask.pocoo.org/
-  [SQLite]: https://www.sqlite.org/
-  [python-requests]: http://docs.python-requests.org/en/master/
-  [Git]: http://git-scm.com/
-  [GitHub]: https://github.com/
-  [mailing list]: https://groups.google.com/a/numfocus.org/forum/#!forum/gsoc
+[Software Carpentry]: https://software-carpentry.org/
+[gsoc-issues]: https://github.com/numfocus/gsoc/issues
+[swc]: https://github.com/swcarpentry/workshop-template/tree/gh-pages/setup
+[swc-install]: https://github.com/wking/swc-setup-installation-test
+[Python]: https://www.python.org/
+[Django]: https://www.djangoproject.com/
+[AMY]: https://github.com/swcarpentry/amy
+[Flask]: http://flask.pocoo.org/
+[SQLite]: https://www.sqlite.org/
+[python-requests]: http://docs.python-requests.org/en/master/
+[Git]: http://git-scm.com/
+[GitHub]: https://github.com/
+[mailing list]: https://groups.google.com/a/numfocus.org/forum/#!forum/gsoc
```

Comment:
You are learning from the best! This is something that @wking would say. :-)

> I don't think it's a good change since you're overwriting the history.

I would avoid this in the future but I will keep this for now.